capa/features/extractors/cape/extractor.py

```diff
@@ -26,26 +28,26 @@
 
 logger = logging.getLogger(__name__)
 
-TESTED_VERSIONS = ("2.2-CAPE",)
+TESTED_VERSIONS = {"2.2-CAPE", "2.4-CAPE"}
 
 
 class CapeExtractor(DynamicFeatureExtractor):
-    def __init__(self, cape_version: str, static: Dict, behavior: Dict):
+    def __init__(self, report: CapeReport):
         super().__init__()
-        self.cape_version = cape_version
-        self.static = static
-        self.behavior = behavior
+        self.report: CapeReport = report
```

Comment:
passing around the `CapeReport` type lets mypy alert us when we access invalid properties.

we don't have quite the same shortcuts (`static` and `behavior`) as before, but they're trivial to access in the CapeReport.